robot-server/robot_server/runs/run_data_manager.py

```diff
@@ -357,7 +357,7 @@ async def update(self, run_id: str, current: Optional[bool]) -> Union[Run, BadRu
             parameters = self._run_orchestrator_store.get_run_time_parameters()
             run_resource = self._run_store.get(run_id=run_id)
 
-        self._runs_publisher.publish_runs_advise_refetch(run_id)
+        await self._runs_publisher.publish_runs_advise_refetch_async(run_id)
```

Comment:
```suggestion
        self._runs_publisher.publish_runs_advise_refetch_(run_id)
```
Are we sure on this one? I thought all the actual publishing is done synchronously now.